### ISSUE TITLE
Replacing data files with json

### ISSUE THREAD

**Issue by Eruyome** [aRTy42/POE-ItemInfo#63]
I never liked using all those data files for affixes (tiers) and as far as I know there's no script to update/compile this data.

Maybe using this here is worth a shot https://github.com/brather1ng/RePoE, would require some work though, replacing all that file parsing code presently used. We could either directly use the uploaded data files there or use the same tools to generate this data from the GGPK, maybe further processing it.

I never tried touching all this parsing from the GGPK because I don't really know python and don't like it after trying to do some basic things here and there :p

**Comment by Eruyome** [comment]
I didn't like to look at a 3mb big json file, so I filtered the unique mods https://github.com/Eruyome/poe-filterItemAffixes/tree/master/output. I think that data is pretty nice.

**Comment by aRTy42** [comment]
The current data files are not a very sophisticated solution, but they are easy to access, understand and edit. Since the affix brackets don't change that often (or ever), I don't really see the benefit of replacing everything.

Furthermore, the internal names of the mods seem slightly annoying to handle, with some having an extra underscore at the end for example. I'd bet that making sure that the script handles every mod correctly is not so straightforward and nice in the end. If we processed the data even further to bundle mod brackets together, we are getting close to what we have now.

So, basically: What do we actually gain?

**Comment by Eruyome** [comment]
Data that doesn't have to be updated manually :p 
I know this can be a lot of work, I'm just saying it's a possiblity. I'm probably a bit biased because I initially wanted to show the min/max values possible for rare items (advanced search), ignoring tiers. I mostly didn't do it yet because I didn't have the affix data and didn't want to use the files from the ItemInfo data folder ;-)

**Comment by Eruyome** [comment]
Oh btw, those names (ids) can be translated easily https://raw.githubusercontent.com/brather1ng/RePoE/master/data/stat_translations.json. Just saying.

https://github.com/brather1ng/RePoE/blob/master/docs/mods.md

**Comment by aRTy42** [comment]
It was not about translation, it is about correctly putting together what belongs together. For example having "ChanceToIgniteJewel_" with underscore but "ChanceToFreezeJewel" without or having "ColdDamagePercentEssence... 1, 2, 3, 4_, 5, 6_, 7". What currently sits in a single txt file is not grouped in those json files, two brackets of the same mod are stored structurally identical to a bracket of a different mod. Entries do have the key "group", but those are broader than our txt files. I see no trivial way to use the file as it is now nor a trivial way to further process it to our needs. Of course it is possible, but as I wrote above, the data barely changes, so manually updating is a non-issue.

**Comment by Eruyome** [comment]
I'm not sure if you get any notification (issue reference) but it might interest you.
